registry

```diff
@@ -524,3 +524,46 @@ erm:ERM00000506 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I1" ;
 erm:ERM00000507 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I2" ; dct:creator wd:Q27061849 .
 erm:ERM00000508 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I3" ; dct:creator wd:Q27061849 .
 erm:ERM00000509 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material 14" ; dct:creator wd:Q27061849 .
+
+
+erm:ERM00000510 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 1" ; dct:creator wd:Q27061849 .
```

Comment:
@vedina, I updated the title of the issue. Or are the labels wrong?

Reply:
no, there are both HARMLESS and SABYDOMA materials

Reply:
Okay, clear!